okapi-core/src/test/java/org/folio/okapi/ProxyTest.java

```diff
@@ -314,14 +342,15 @@ private void td(TestContext context, Async async) {
   @After
   public void tearDown(TestContext context) {
     Async async = context.async();
-
-    HttpClientLegacy.delete(httpClient, port, "localhost", "/_/discovery/modules", response -> {
-      context.assertTrue(response.statusCode() == 404 || response.statusCode() == 204);
-      response.endHandler(x -> {
-        httpClient.close();
-        td(context, async);
-      });
-    }).end();
+    httpClient.delete(port, "localhost", "/_/discovery/modules",
+        MultiMap.caseInsensitiveMultiMap(), res1 -> {
+          context.assertTrue(res1.succeeded());
```

Comment:
not sure it's a big deal, but is this really an equivalent test to 404 or 204? <thinking>

Reply:
Adding a test for 204 there.